examples/website-demos/virtual-event/src/components/CentralPanel/EventCard.scss

```diff
@@ -0,0 +1,84 @@
+@import '../../styles/variables';
+
+.event-card {
+  &-container {
+    display: flex;
+    overflow: hidden;
+    box-shadow: 0px 0px 8px rgba(0, 0, 0, 0.15);
```

Comment:
i believe this color should be a variable

Reply:
No variable matches